packages/firestore/src/remote/remote_store.ts

```diff
@@ -773,14 +761,13 @@ export class RemoteStore {
       this.writeStream.lastStreamToken = emptyByteString();
 
       return this.localStore.setLastStreamToken(emptyByteString());
-    } else {
-      // Some other error, don't reset stream token. Our stream logic will
-      // just retry with exponential backoff.
-      return Promise.resolve();
     }
```

Comment:
Can you leave the else block (just containing the comment)? It makes the context for the comment more clear.